test/functional/feature_assumeutxo.py

```diff
@@ -81,7 +81,7 @@ def run_test(self):
         self.sync_blocks()
 
         def no_sync():
-            pass
+            self.no_op
```

Comment:
It was actually better to remove no_sync completely and just use `self.no_op` directly. I think it's clearer then because we don't need the same no_op function declaration that we already have in the framework.